client/src/Pages/History/History.js

```diff
@@ -1,7 +1,7 @@
-import React,{useEffect} from 'react';
+import React,{useEffect, useState} from 'react';
```

Comment:
Not for this part but for the whole code, the header in your app is almost similar to the body of the app, try to make changes in its CSS so that it becomes a bit noticeable, and also you can try to make your header sticky. It's convenient for the user. Just a suggestion

Reply:
okay will work on it thanks for your suggestion